charters/DataOperations/charter.md

```diff
@@ -0,0 +1,102 @@
+  - Monitor the status of data contributions at each step in the DCP data lifecycle from initial project notification and shepherd all the way through release and potential inclusion on tertiary portals and/or in additional data distribution
+  - Identify processes in the data lifecycle that require improved monitoring, streamlined troubleshooting, automation, etc. towards a more scalable and efficient data lifecycle with increased data contribution bandwidth
+  - Engage with Analysis Work Group to identify data quality control standards for each data distribution
+  - Engage with all DCP components to ensure proper checks of data quality standards are put in place within the appropriate components throughout the data lifecycle
+  - Engage with the [Compliance Working Group](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md) to ensure the our data release and distribution processes ensure responsible HCA data sharing to all types of consumers
+  - Ensure released data that are used on tertiary portals can be easily cross-reference across portals
+  - Determine best practices for when and how changes are made to DCP outputs after release
+
+- Community engagement
+  - Define data use policy and citation instructions for HCA data users
+  - Define mechanism(s) for which the community is notified when DCP products are updated after being released
+  - Partner with Content and Portal Development to ensure clear, findable, and current documentation describing the data lifecycle and data distribution
+  - Ensure effective communication of HCA data updates and user features to the scientific community
+  - Partner with UX to assess the added value of data distribution through communities’ feedback, and better understand their expectations and needs
+  - Onboard staff to DCP ZenDesk ticketing system
+  - Take ownership of establishing process and interfacing with DCP components for assigning, responding, and integrating ZenDesk tickets
+  - Compile feedback gained from consumers through ZenDesk queries and comments, and report back to the DCP in order to facilitate system improvement 
+
+
+### Scientific "guardrails"
+
+In order to meet the needs of the communities of scientific consumers, the DataOps team requires input from the Analysis Working Group as to where curated data distribution will add value for the researcher.
+
+## Out-of-scope
+
+The Data Operations is not responsible for determining what data or data types are made available through the DCP.
+
+As many of the features and requirements for the data lifecycle and data distribution will be implemented through other components, the majority of technical aspects pertaining to the data lifecycle and data distribution will be implemented by the appropriate component, and not Data Operations.
```

Comment:
In line with my above comment, this handoff of requirements to implementation teams, and then back again for testing and validation, has a major risk of generating (more, different) silos as well as generating costly handoffs requiring high bandwidth communication to work through. Can you explain how you will de-risk this?

Reply:
I see this as DataOps owning proposals, laying out functional specifications & collaborating with various components as questions arise during implementation. And DataOps verifying the finished product addresses the intended use. The project completion RFC is the first example of this.
Proposal goes out and if it gets approved, then it gets lined up in the Roadmap and in the work queue of one or more relevant components without consideration of who tossed it out there to begin with.

Reply:
I'm happy with this if the de-risking strategy is to publish RFCs when it handoffs between teams are necessary. I know there are other efforts ongoing aimed at improving the RFC process and it's reasonable to let that play out independently

Reply:
Yes, we intend to use the existing framework in the DCP, as it currently is and as it evolves
(thank you for not asking me to put 'own DCP-wide communication and decision-making procedures' on our charter!)